sale_commission/views/settlement_view.xml

```diff
@@ -55,35 +55,40 @@
                                 name="action_cancel" />
                         <field name="state" widget="statusbar"/>
                     </header>
-                    <h1><field name="agent" /></h1>
-                    <group>
+                    <sheet string="Settlement">
```

Comment:
Why these changes? This was recently changed in https://github.com/OCA/commission/pull/92, so you shouldn't remove it